MesaModel/model/Pupil.py

```diff
@@ -561,20 +560,20 @@ def changeState(self):
                 self.model.model_state.learning_count -= 1
             return 1
 
-    def set_start_math(self):
+    # Calculate the end maths score based on the time spent by a pupil in
+    # a learning state and the pupil's starting maths score
```

Comment:
Worth adding more detail here about how the calculation is done, because it's different to the NetLogo model.